ethers-utils/src/schema.graphql

```diff
@@ -0,0 +1,15 @@
+type Module {
+  solidityKeccak256Bytes(bytes: String!): String!
```

Comment:
This method has bytes in the name but accepts a string argument and returns a string. Maybe adjust the name? Same with other methods